wagtailio/static/sass/components/_modal.scss

```diff
@@ -0,0 +1,154 @@
+.modal {
+    $root: &;
+
+    &--get-started {
+        // hide the modal if it's open below medium breakpoint
+        &.micromodal-slide.is-open {
+            display: none;
+
+            @include media-query(medium) {
+                display: block;
+            }
+        }
+    }
+
+    &__overlay {
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
```

Comment:
Suggestion: Could we use `inset` for this instead?

Reply:
Definitely - much more concise! : )